client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -149,8 +149,8 @@ function EmailPlan( { domain, hideHeaderCake = false, selectedSite, source } ) {
 			};
 		}
 
+		// There is no "disabled" configuration since it's allowed adding/removing forwarding for every site admin
```

Comment:
```suggestion
		// All site admins are allowed to add/remove email forwards
```
Rephrasing the comment so it's clearer :)